components/enterprise-contract/base/chains-config.yaml

```diff
@@ -3,6 +3,9 @@ kind: ConfigMap
 metadata:
```

Comment:
Are we sure we want to move the Chains configuration from the pipeline-service to the enterprise-contract? It seems a little odd, and confusing, to have the Deployment and the configuration split like this.

Reply:
Can’t manage it otherwise, would lead to having two applications compete over it. I tried `replacements`, that works only within the same kustomization. The idea there was to have a ConfigMap in `enterprise-contract-Service` and lookup values from `tekton-chains`, that does not seem possible.